docs/pages/deploy-a-cluster/helm-deployments/digitalocean.mdx

```diff
@@ -246,7 +246,7 @@ guide](../../reference/helm-reference/teleport-cluster.mdx).
 Read our guides to additional ways you can protect a Kubernetes cluster with
 Teleport:
 
-- [Connect Multiple Kubernetes Clusters](../../kubernetes-access/register-clusters/register-via-deployment.mdx)
+- [Connect Multiple Kubernetes Clusters](../../kubernetes-access/getting-started.mdx)
```

Comment:
```suggestion
- Connect another Kubernetes cluster to Teleport by [deploying the Teleport Kubernetes Service](../../kubernetes-access/getting-started.mdx)
```
Suggested to make this more precise as to the content of the guide we're linking to